### DISCUSSION TITLE
Try to observe changes in local storage

### DISCUSSION THREAD

**Discussion by KRav007** [cyrilletuzi/angular-async-local-storage#273] [Q&A] [answered]
I subscribe like this to the value in local storage, but if I change the value on a second tab and come back to the main-page, ,there is no console log, that says 'new value'.
I understand the subscribe as an observable and I expect that it listens to the localstorage. Is this right? 

 this.storage.get('survey_submitted').pipe(
      catchError(() => of('survey_submitted')),
    ).subscribe((result) => {

      console.log('local storage changed');
      console.log(result);

    });

**Comment by cyrilletuzi** [comment, marked as answer]
As stated in the README:

> Note you'll only get one value: the Observable is here for asynchrony but is not meant to emit again when the stored data is changed. And it's normal: if app data change, it's the role of your app to keep track of it, not of this lib. See #16 for more context and #4 for an example.

A `watch()` method has been added in v9 (see #108).

**Comment by KRav007** [comment]
Ok thank you
